source/compatibility.tex

```diff
@@ -1922,6 +1922,34 @@
 
 \rSec2[diff.cpp17.dcl.dcl]{\ref{dcl.dcl}: declarations}
 
+\diffref{dcl.typedef}
+\change Unnamed classes with a typedef name for linkage purposes
+can only contain C-compatible constructs.
+\rationale Necessary for implementability.
+\effect Valid C++ 2017 code may be ill-formed in this International Standard.
+\begin{codeblock}
+typedef struct {
+  void f() {}           // ill-formed; previously well-formed
+} S;
+\end{codeblock}
+
+\diffref{dcl.fct.default}
+\change A function cannot have different default arguments
+in different translation units.
+\rationale Required for modules support.
+\effect Valid C++ 2017 code may be ill-formed in this International Standard,
+with no diagnostic required.
+\begin{codeblock}
+// Translation unit 1
+int f(int a = 42);
+int g() { return f(); }
+
+// Translation unit 2
+int f(int a = 76) { return a; } // ill-formed (no diagnostic required); previously well-formed
+int g();
+int main() { return g(); }      // used to return 42
```

Comment:
... however, these comments should be moved to multiples of 4 to follow our  guidelines.

Reply:
These are at col 32. I've moved them to col 40 to satisfy the "at least two spaces" rule.